testerum-backend/testerum-runner/testerum-runner-junit/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>testerum-runner</artifactId>
+        <groupId>com.testerum</groupId>
+        <version>develop-SNAPSHOT</version>
+    </parent>
+
+<!--  TODO IONUT: rename this module  -->
+    <artifactId>testerum-runner-junit</artifactId>
+    <packaging>jar</packaging>
+
+
+    <dependencies>
+        <!--==================== Internal Dependencies ====================-->
+        <!-- testerum-runner-cmdline -->
+
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>testerum-runner-cmdline</artifactId>
+        </dependency>
+
+        <!-- STEPS -->
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>assertion-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>debug-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>http-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>json-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>rdbms-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>selenium-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>util-steps</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.testerum</groupId>
+            <artifactId>vars-steps</artifactId>
+        </dependency>
+
+        <!--==================== External Dependencies ====================-->
+
+        <!-- Kotlin -->
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-reflect</artifactId>
+        </dependency>
```

Comment:
Do we need ``kotlin-reflect``? I don't think so...

Reply:
I don't know, they recommend it on how to start a Kotlin project. I don't debate